appinventor/components/src/com/google/appinventor/components/runtime/Form.java

```diff
@@ -2393,6 +2397,20 @@ private void showAboutApplicationNotification() {
     Notifier.oneButtonAlert(this, message, title, buttonText);
   }
 
+  public void getComponent(View component,ContextMenu menu){
```

Comment:
Space after `,`

Is there a better way to manage this without having to store a map of every component to every menu? For example, could a component register for a menu and each menu store its corresponding components as a set?